dashboard/config/levels/custom/dance/allthethings HOC Dance 1.level

```diff
@@ -10,6 +10,7 @@
       "DanceLab"
     ],
     "hide_animation_mode": "true",
+    "uses_preview" : "true",
```

Comment:
I wonder if we should leave the first sample level using the legacy behavior, and apply this to a later level in `allthethings`?

Reply:
Yes - that makes sense. Asking a question in [this PR](https://github.com/code-dot-org/code-dot-org/pull/53769/files#r1330241656) about where to add new level